packages/sit-onyx/src/components/OnyxButton/types.ts

```diff
@@ -35,6 +35,10 @@ export type OnyxButtonProps = DensityProp &
      * Whether to show a skeleton button.
      */
     skeleton?: boolean;
+    /**
+     * Whether to enable ripples on interaction
+     */
+    rippleAnimation?: boolean;
```

Comment:
The button should always have the ripple animation, so we don't need to provide a property to disable it

Reply:
should I also remove the ripple-animation css class?

Reply:
Hint: should be disabled when the button is disabled

Reply:
> should I also remove the ripple-animation css class?

If its no longer needed yes :)